releasenotes/releasenotes.rst

```diff
@@ -24,6 +24,55 @@ Public Endpoint Announcement 8 July 2021
 
 The default FHIR version of the `public Firely Server endpoint <https://server.fire.ly/>`_ is now R4.
 
+.. _vonk_releasenotes_470:
+
+Release 4.7.0, Feb 1st, 20222
+-----------------------------
+
+.. attention::    
+    With version 4.7.0, Firely Server migrated to .NET 6.0. In order to run the binaries, `ASP.NET Core Runtime 6.x <https://dotnet.microsoft.com/en-us/download/dotnet/6.0>`_ needs to be installed.
+
+
+Feature
+^^^^^^^
+
+#. BulkDataExport is now supported for MongoDB as well. Get started with the :ref:`Bulk Data Export documentation<feature_bulkdataexport>`.
+#. Circular references in transaction bundles are now supported. Bundles of type ``transaction`` and ``batch`` are permitted to contain resources referencing another resource within the same bundle. This also means that you can now cross reference ``PUT`` and ``POST`` entries.
+#. An option to configure additional token issuers is now available. This is used in settings where the token issuer deviates from the token audience. This new setting replaces the existing ``AdditionalBaseEndpoints``. The setting needs to be adjusted manually as it will not be migrated automatically.
+#. Firely Server now supports receiving document bundles on the transaction base endpoint. Firely Server will extract the narrative of document bundles and store this within a DocumentReference resource. 
+#. Added support for transforming SMART scopes issued by Azure Active Directory.
+#. Firely Server will now recognize the ``name`` claim in JSON Web Tokens and also include its content in the logs.
+#. It is now possible to provide the Firely Server license via an environment variable.
+
+Plugins
+^^^^^^^
+
+.. attention::    
+    Please ensure that your plugins for Firely Server are targeting at least ``net6.0``. Plugins that are targeting ``netstandard2.0`` or ``netstandard2.1`` do not need to be updated.
```

Comment:
This is somewhat confusing to me. Shouldn't it state that plugins that target netstandard 2.0/2.1 or netcore3.1 do not need to be updated and that new plugins can now also (optionally) target net6.0?

Reply:
Agree.

Reply:
Actually, we can remove the entire warning. Nothing changes for plugin creators as they are targeting netstandard xx anyway, which is compatible with .NET 2.0 and above: https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0